Task7/stringRules/MoreThanFiveWords.java

```diff
@@ -0,0 +1,32 @@
+package stringRules;
+
+/**
+ * extended from Rule class that 
+ * specifies rule №3 "Contains more that five words"
+ * @param line is an original line that must be
+ * checked to compliance to described rule
+ */
+public class MoreThanFiveWords extends Rule {
+
+    String line;
+    
+    public MoreThanFiveWords(String line) {
+        this.line = line;
+    }
+    
+    /**
+     * checks if line has more than five words
+     * @param words is the array of separate words from
+     * original line
+     * @return string message about compliance if it does
+     */
+    @Override
+    public String checkRule() {
+        String[] words = intoWordsArray(line);
+        if (words.length>5) {
+            return ("Complies rule 'Has more than 5 words' \n");
+        } else {
+            return null;
```

Comment:
Try to avoid returning null - it will cause NullPointer exception if caller doesn't check result for null